poc_iot_verifier/src/runner.rs

```diff
@@ -196,7 +202,9 @@ impl Runner {
                 Some(res) => res,
                 None => {
                     tracing::debug!("beacon verification failed, reason: EntropyNotFound");
-                    return Ok(());
+                    _ = Report::update_attempts(&self.pool, &beacon_report.ingest_id(), Utc::now())
+                        .await;
```

Comment:
if we're not logging here each time we fail to validate the report are we logging the hash or timestamp or some other identifier for all of the reports when they finally get purged from the DB at max_failed_attempts?